index.html

```diff
@@ -144,7 +144,7 @@
     right around the corner of this tag ;)
     
 -->
-<div id="impress">
+<div id="impress" data-hide-mouse-after="3000">
```

Comment:
Hi sclausen

Good idea and easy to implement.

IMO this feature needn't be optional. (Not adding an option benefits the goal of simplicity.) I don't think anyone will miss the pointer if we just hide it always after 3 seconds of inactivity. (I can see that someone would want to leave the mouse pointer pointing at something on a slide. But that should be rare. You'd usually move the pointer to point at something, but that would still work. Also, it is more common to use laser pointers, etc.)